### PR TITLE
chore: general cleanup

### LINKED ISSUE #38: Change package name
Change perfect-next-theme to fiveable-open.

### LINKED ISSUE #39: Styling on mdx layout
- reduce padding in the x direction on mobile
- line height on `<h2>` is too large on mobile
- make links a different color and underline to make it clear that it's a link

### PR DESCRIPTION
**Pull request by zaida04**
This PR aims to address various style/structural issues/inconsistencies across the project as a whole. This will include website content changes, but can also include meta stuff too.

### CODE REVIEW

**Comment by awlevin** [comment]
#### 📝Changed routes:


| [**/about**](https://open-ate0ap3gf-thinkfiveable.vercel.app/about) | [**/**](https://open-ate0ap3gf-thinkfiveable.vercel.app/) |
|:-:|:-:|
|<a href="https://open-ate0ap3gf-thinkfiveable.vercel.app/about">Screenshot of /about</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/M7OQ9r7QAn2kbRgR-pgtyUpQJCHhYgLf2qDIWTAmnIVkD5KmZ2_cV03H2Wk0Wwq6jtBfoX9NkQ_VAy-IXe0iLw.png">(view full size)</a> |<a href="https://open-ate0ap3gf-thinkfiveable.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/yDYVJP1YjIjvF5fzYJazO0nfGBgmWS3whNCIQCl3dqCe0dTHK4npjHSOh6xc8jpMT_cWYhdjeM69X0AjtcjNLw.png">(view full size)</a> |







Commit 0a02a150a688c84c6d5346a17b808348c0c8a724 (https://open-ate0ap3gf-thinkfiveable.vercel.app).

**Comment by exu3** [comment]
fixes #39

**Review by exu3 — DISMISSED** [review]

File: pages/about.mdx [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import Layout from "../components/layout";
+import Footer from "../components/footer";
 
 export default (children) => (
-    <Layout title="About" subtitle="Fiveable Open Source Initiative" {...children} />
+    <div>
+        <Layout title="About" subtitle="Fiveable Open Source Initiative" {...children} />
```
Comment (sarthaktexas) [inline pages/about.mdx:6]: was this done by the linter?

File: pages/index.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ export default function Home() {
                         We&#39;re building a community for discovery and learning.
                     </h2>
                     <Link href="/about">
-                        <a className="font-rubik font-medium bg-black text-center rounded-full px-5 py-1 text-lg text-white w-fitcontent flex flex-row cursor-pointer">
+                        <a className="md:animate-bounce hover:text-gray-300 font-rubik font-medium bg-black text-center rounded-full px-5 py-1 text-lg text-white w-fitcontent flex flex-row cursor-pointer">
```
Comment (sarthaktexas) [inline pages/index.js]: i don't support the animate-bounce here at all

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

File: pages/index.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ export default function Home() {
                             </div>
                         </div>
                         <a
-                            className="md:hidden hover:text-gray-500 absolute top-4 right-4 sm:top-8 sm:right-8 p-4 bg-white rounded-full"
+                            className="md:animate-pulse md:hidden hover:text-gray-500 absolute top-4 right-4 sm:top-8 sm:right-8 p-4 bg-white rounded-full"
```
Comment (sarthaktexas) [inline pages/index.js]: the pulse animation doesn't work i think

Comment (sarthaktexas) [reply pages/index.js]: Also that's not what pulse is supposed to be used for iirc

File: pages/about.mdx (hunk 1 above)
Comment (zaida04) [reply pages/about.mdx:6]: Nah, originally the footer was imported in this file before we added it to layout.js, and for me to use it with this Layout tag I had to wrap the whole thing in a div. We can just leave this tbh, incase we need it in the future.

File: pages/index.js (hunk 3 above)
Comment (zaida04) [reply pages/index.js]: resolved in 223530c

**Review by exu3 — DISMISSED** [review]

File: pages/index.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ export default function Home() {
                         We&#39;re building a community for discovery and learning.
                     </h2>
                     <Link href="/about">
-                        <a className="font-rubik font-medium bg-black text-center rounded-full px-5 py-1 text-lg text-white w-fitcontent flex flex-row cursor-pointer">
+                        <a className="hover:text-gray-300 font-rubik font-medium bg-black text-center rounded-full px-5 py-1 text-lg text-white w-fitcontent flex flex-row cursor-pointer">
```
Comment (sarthaktexas) [inline pages/index.js]: why don't we do a translate on hover cause changing text color isn't accessible

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

**Review by sarthaktexas — APPROVED** [review]

**Review by exu3 — APPROVED** [review]

**Pull request merged**